UPDATE.md

```diff
@@ -0,0 +1,56 @@
+# Update cloud-provider-gcp with Kubernetes release
+
+## Overview
+
+k8s.io/cloud-provider-gcp provides a sample distribution of how to run Kubernetes on the Google Cloud.
+This project allows a Kubernetes cluster to provision, monitor and add/remove GCP resources necessary for operation of the cluster. 
+It has to be updated together with Kubernetes releases. 
+This document describes the steps to update cloud-provider-gcp with Kubernetes release.
+
+## Workflow
+
+1. Update library to the latest version.
```

Comment:
Would it make sense to list the libraries are currently aware of?

Reply:
It might change along the way. Since we specify the update is for a desired kubernetes release version, it should be clear about the libraries need to be updated.